src/main/java/core/basesyntax/service/impl/ReaderFileServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class ReaderFileServiceImpl implements ReaderService {
+    private static final int INDEX_TITLE_LINE = 0;
```

Comment:
we should be able to read data from any file, now you always skip the first line

let's skip it in main() or in parser